interface/main/calendar/config.php

```diff
@@ -12,8 +12,15 @@
 //                  1 for encoded
 // ----------------------------------------------------------------------
 //
-include_once("../../../library/sqlconf.php");
-// Modified 5/2009 by BM for UTF-8 project 
+
+// to collect sql database login info and the utf8 flag
+// also collect eh adodb libraries to support mysql_log that is needed for mysql ssl support
```

Comment:
typo fix:
``````
// also collect the adodb libraries to support mysqli_mod that is needed for mysql ssl support